spec/table_spec.rb

```diff
@@ -2792,6 +2790,24 @@
         end
       end
     end
+
+    context "when `except` is passed" do
+      it 'resizes only the other columns' do
+        # Check that an Array argument works
+        table = Tabulo::Table.new(%w(hello hi there), :to_s, :length, :upcase)
+        expect { table.pack(except: [:to_s, :length]) }
+          .to change(table.column_registry[:to_s], :width).by(0)
+          .and change(table.column_registry[:length], :width).by(0)
+          .and change(table.column_registry[:upcase], :width).to(6)
+
+        # Check that a single argument works
+        table = Tabulo::Table.new(%w(hello hi there), :to_s, :length, :upcase)
```

Comment:
Please stick to square brackets for `%w[these kinds of arrays]`, for consistency with the project's preexisting style.

Reply:
Done.